app/views/groups/new.html.erb

```diff
@@ -0,0 +1,16 @@
+ <% if user_signed_in? %>
+      <%= render 'partials/navbar'  %>
+    <% end  %>
+
+<div class="categories" style="width:100%">
+
+  <h3 class="mx-5">Add New Category</h3>
+
+  <%= render "form", group: @group %>
+
+  <br>
+
+  <div>
+    <%= link_to "Back to groups", user_groups_path %>
+  </div>
+</div>
```

Comment:
- [ ] Great job so far 👍🏻 However the `Back to group` button should be a back icon placed in the header navbar. Kindly address this 

Your Implementation | Original Design
:---:|:---:
![image](https://user-images.githubusercontent.com/79058364/234993167-9f2db0a8-c92c-45e0-9a18-8fd879c5e045.png) | ![image](https://user-images.githubusercontent.com/79058364/234993723-2af7342c-31de-472a-9c98-6faf479fa494.png)